app/components/avo/fields/show_component.rb

```diff
@@ -61,4 +61,8 @@ def field_wrapper_args
   def disabled?
     field.is_readonly? || field.is_disabled?
   end
+
+  def copyable?
+    field.respond_to?(:copyable) && field.copyable
```

Comment:
That is solution but only fields with text should have option to be copyable and only in show and index view

Reply:
Yeah, my point is to try to avoid usage of `respond_to?` which is a code smell. You are already trying to ask every field if its copyable of not so you could as like allow every fileld to be able to respond to this msg (and return false by default for example by impelemnting this method in some base field and then overriging it with true where it apply)

Reply:
I agree with @KamilMilewski.  

Let's insert the `copyable` property on the base field instead of adding it to each individual field we want to support. Set the default value to `false`, allowing us to simply check `field.copyable`.  

This approach reduces complexity for now.